.github/CODEOWNERS

```diff
@@ -548,6 +548,12 @@
 /core/templates/services/services.constants.ajs.ts @vojtechjelinek
 /core/templates/services/services.constants.ts @vojtechjelinek
 
+
+# Dynamic Feature Gating project
+/core/storage/platform_parameter/ @MegrezZhu
```

Comment:
For storage models it can fit in the broader umbrella, but for domain objects 'core/domain/platform_parameter*`, looks like there isn't an entry covering all domain level files, how should we handle the domain level files then?

Reply:
I suggest adding me and @vinitamurthi as code owners to start. You can also add us as code owners for the storage model. I didn't realize that core/domain had this inconsistency, so we should ensure the owners for this feature are consistent for the full implementation.

Reply:
Done, added @BenHenning and @vinitamurthi as the code owners of  `core/domain/platform_parameter*`.
Also, since the storage models were moved to `code/storage/config/` whose code owner is @DubeySandeep, to keep the consistency across all feature-gating-related files, I also added both @BenHenning @vinitamurthi as the code owner of `code/storage/config/`.